### ISSUE TITLE
ability to specify profile to use from bookmark/url

### ISSUE THREAD

**Issue by MarianneLHyman** [hultmann/multifox#224]
It would be very useful to be able to specify which profile to use for a bookmark, for example.

Let's say the user has bookmarked example.com ... the "Location" field for this bookmark would normally read as follows:
http://example.com/

I am proposing that the following special values (or similar) be recognized by Multifox:
multifox-default:http://example.com/
multifox-new:http://example.com/
multifox-named:http://example.com/ (where "named" is replaced by the name of a particular profile)

This would also allow users to decide which profile should be used when they open a new tab, by setting browser.newtab.url to such as:
multifox-default:about:blank
multifox-new:about:blank

It may be useful to reference how this was accomplished in the "Private Tab" extension (https://addons.mozilla.org/en-US/firefox/addon/private-tab/)

**Comment by benoon** [comment]
Yes, agree with Marianne, ability to assign a profile to bookmark, that would make us clicking less :)
